src/javascript/app_2/App/Components/Elements/Media/index.js

```diff
@@ -0,0 +1,4 @@
+export *       from './MediaHeading.jsx';
+export default from './MediaItem.jsx';
+export *       from './MediaIcon.jsx';
+export *       from './MediaDescription.jsx';
```

Comment:
aren't we supposed to name them like `media-description.jsx`? referring to other index.js in adjacent folders

Reply:
Yes. My bad.